portfolio/.project

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
we have some hidden files still in the portfolio/ directory. do we still need them?

Reply:
We deleted the portfolio folder because we don't need it. Also we decided that we're going to delete the Angular app and put the components in again since we're running into node module errors